src/redux/initialState.js

```diff
@@ -1251,6 +1251,156 @@ const initialState = {
       newFurniture: true,
       hotDeal: true,
     },
+    {
+      id: 'aenean-ru-bristique-84',
+      name: 'Aenean Ru Bristique 84',
+      category: 'furniture',
```

Comment:
nie ma kategorii furniture. Furniture to wszystkie meble. Nie musisz dodawać nowych kategorii. Wszystko już jest w initialState